language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/CommandSubmissionClientImpl.java

```diff
@@ -39,7 +39,20 @@ public CommandSubmissionClientImpl(
         StubHelper.authenticating(CommandSubmissionServiceGrpc.newFutureStub(channel), accessToken);
   }
 
-  public Single<com.google.protobuf.Empty> submit(
+  @Override
+  public Single<Empty> submit(CommandsSubmission params) {
```

Comment:
```suggestion
  public Single<Empty> submit(CommandsSubmission submission) {
```

Minor, to your discretion. To be applied here and other instances.